arrow/locales.py

```diff
@@ -95,15 +147,21 @@ def describe(self, timeframe, delta=0, only_distance=False):
 
         return humanized
 
-    def describe_multi(self, timeframes, only_distance=False):
+    def describe_multi(
+        self,
+        timeframes: Collection[Iterable[Union[TimeFrames, int, float]]],
```

Comment:
arrow do not handling length of a element of `timeframes`. So it will be good if we can set the type of `timeframes` to `Collection[Tuple[TimeFrames, Union[int, float]]]`. but I could not to set like that because currently there're no restriction for type of `timeframes` elements.